src/terraform/command_helpers.go

```diff
@@ -34,6 +34,7 @@ func compileEnvVars(deployConfig types.DeployConfig, secrets types.Secrets) ([]s
 		for _, secretKey := range serviceSecrets {
 			serviceEnvVars[secretKey] = secrets[secretKey]
 		}
+		serviceEnvVars["ROLE"] = serviceName
```

Comment:
`ROLE` is an ExoCom specific env var, it probably shouldn't be passed to every service.

Reply:
We'll have to do something similar where `ROLE` is set only when `appDependency` is `exocomDependency`

Reply:
`ROLE` is passed to everything in the local environment: https://github.com/Originate/exosphere/blob/v0.23.0.alpha.4/src/docker/composebuilder/index.go#L40

Its also not exocom specific to me. Its the name given the service in the `application.yml` and would be there whether or not you are using exocom. The role allows you to give labels to a service that may be a configurable reusable service.

Reply:
Hm that must've slipped through a PR.

Yes it's true that we can use `ROLE` for anything, but the reason we set that variable in the first place is so exoservice/relay has a name to give Exocom when it registers. It's technically not needed if you're not using Exocom. I'm fine with leaving it as is, but that just means we'll always be defining the `ROLE` env var for the user and that'll be a variable name they can't use.

Reply:
I think its a good generic one for now. We can certainly make it exocom specific if ever is annoying to someone. I'll update this so similar to in the local version where it is assigned first so it could be overridden.